grants_tagger_light/training/train.py

```diff
@@ -33,6 +38,11 @@ def train_bertmesh(
     data_path: str,
     training_args: TrainingArguments,
     model_args: BertMeshModelArguments = None,
+    max_samples: int = -1,
+    test_size: float = 0.05,
+    num_proc: int = os.cpu_count(),
+    batch_size: int = 256,
```

Comment:
I think this is the reason you are going fast, this is 4x larger than previous but you still might need 4x the steps to achieve similar performance, we can experiment with this

Reply:
Good point. Let's see, definitely with all the changes we need to experiment now with the hyperparams, it may have changed a lot.